util: add support for `maxArrayLength` when inspecting `Set` and `Map` (#43576)
nodejs/node: doc/api/util.md

<!-- YAML
added: v0.3.0
changes:
  - version: REPLACEME
    pr-url: https://github.com/nodejs/node/pull/43576

>     description: add `maxArrayLength` option to `Set` and `Map`.

Reviewer: ```suggestion
    description: add support for `maxArrayLength` when inspecting `Set` and `Map`.
```
